data/types/settings/pump/units.go

```diff
@@ -9,6 +9,7 @@ import (
 const (
 	CarbohydrateExchanges = "exchanges"
 	CarbohydrateGrams     = "grams"
+	InsulinUnits          = "Units"
```

Comment:
I can't recall why this was added - can `InsulinUnits` be anything other than `Units` anyway?

Reply:
cc @jamesraby